webapp/components/Comment/Comment.vue

```diff
@@ -86,6 +96,7 @@ export default {
     ContentMenu,
     ContentViewer,
     CommentForm,
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-01-09T07:54:35Z" title="Thursday, January 9th 2020, 8:54:35 am +01:00">Jan 9, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -105,7 +113,7 @@ export default {
       if (this.isLongComment && this.isCollapsed) {
         return this.$filters.truncate(this.comment.content, COMMENT_TRUNCATE_TO_LENGTH)
       }
-
+      // console.log(this.comment.content.replace(/\?/gi, '?++'))
```


Has to be removed later …

Reply:
> [<img alt="ogerly" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/ogerly) **Authored by [ogerly](https://github.com/ogerly)**
_<time datetime="2020-01-15T06:41:28Z" title="Wednesday, January 15th 2020, 7:41:28 am +01:00">Jan 15, 2020</time>_

---


remove